src/Validator/Range.php

```diff
@@ -128,6 +129,11 @@ public function isValid($value): bool
 
         switch ($this->format) {
             case self::TYPE_INTEGER:
+                // Accept infinity as an integer
+                // Since gettype(INF) === TYPE_FLOAT
+                if ($value === INF || $value === -INF) {
```

Comment:
This will overwrite the range check, if a value is INF or -INF still doesn't mean his inside the valid range

Reply:
Addressed in c42cce8